modules/ossm-scoping-service-mesh-with-discoveryselectors.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+// install/ossm-installing-openshift-service-mesh.adoc
+
+:_mod-docs-content-type: Concept
+[id="ossm-scoping-service-mesh-with-discoveryselectors_{context}"]
+= Scoping the Service Mesh with discovery selectors
+:context: ossm-scoping-service-mesh-with-discoveryselectors
+
+A {SMProductShortName} includes workloads that meet the following criteria:
+
+* The control plane has discovered the workload.
+* The workload has an Envoy proxy sidecar injected.
+
+By default, the control plane discovers workloads in all namespaces across the cluster, with the following results:
+
+* Each proxy instance receives configuration for all namespaces, including workloads not enrolled in the mesh.
+* Any workload with the appropriate pod or namespace injection label receives a proxy sidecar.
+
+In shared clusters, you might want to limit the scope of {SMProductShortName} to only certain namespaces. This approach is especially useful if multiple Service Meshes run in the same cluster.
```

Comment:
Also verify "Service Meshes" vs. "service meshes."

Reply:
I will keep this in lower case in here as the product name is not being pointed out in here.